src/sql/workbench/contrib/query/browser/queryResultsView.ts

```diff
@@ -300,43 +300,52 @@ export class QueryResultsView extends Disposable {
 		}
 	}
 
-	public set input(input: QueryResultsInput | undefined) {
-		this._input = input;
-		this.runnerDisposables.clear();
+	private showQueryEditorError(): void {
```

Comment:
please hide the whitespaces to make it easy to tell the actual changes. ![image](https://user-images.githubusercontent.com/13777222/144338556-c533f059-44f3-4b5b-9ee7-c1b083db5002.png)